src/bootstrap/src/core/build_steps/run.rs

```diff
@@ -121,7 +121,6 @@ impl Step for ReplaceVersionPlaceholder {
 
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub struct Miri {
-    stage: u32,
```

Comment:
I realized this field is unnecessary. At least it seems unnecessary, I don't fully understand how this `make_run` vs `run` stuff is meant to be used.